models/src/main/java/com/vimeo/networking2/EditSession.kt

```diff
@@ -18,5 +18,11 @@ data class EditSession(
      * Whether the current version of clip is of max resolution.
      */
     @Json(name = "is_max_resolution")
-    val isMaxResolution: Boolean? = null
+    val isMaxResolution: Boolean? = null,
+
+    /**
+     * Video session id
```

Comment:
There's new detekt rules that require these property docs to be moved to parameter docs. The existing properties were added to the baseline and we've been moving them as we change each file. So this should be move to the class docs as `@param vsid Video session id`. Detekt doesn't require it, but if you have the time to move the docs for the other two properties in this class and then run `./gradlew detektBaseline` that would be appreciated.